lib/pages/portal/view/website_view.dart

```diff
@@ -187,8 +208,61 @@ class _WebsiteViewState extends State<WebsiteView> {
         ],
       );
 
+  // Returns a widget that behaves similarly to a textfield with a "clear"
+  // button, except it actually allows the user to select an image from the
+  // gallery instead of inputting text directly.
+  Widget _uploadButton(BuildContext context) {
```

Comment:
You don't need to pass the context here.  The `State` class already has a `context` you can access directly.